### PR TITLE
Adding door transitions query API.

### PR DESCRIPTION
**Pull request by phil-impinj**

### CODE REVIEW

**Comment by davidyscott** [comment]
Looks good!  I think we're supposed to also bump the version now?

**Comment by phil-impinj** [comment]
Good question on the version bump - @natke ?

**Comment by natke** [comment]
Re: the version, yes, we should bump the version. Until we release the next version to maven central, it should be X.Y.Z-SNAPSHOT.

I would bump the minor version, as this is more than a patch change, so how does 4.1.0-SNAPSHOT sound?

File: src/test/resources/data/ItemDoorTransitionResponse.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "jobId": "051e2ee0-b681-4f62-aff3-5f4d36d60603",
+      "doorId": 2,
+      "destination": "IN",
+      "confidence": 1,
+      "creationTime": "2017-03-01T22:48:15Z"
+    }
+  ],
+  "nextPageMarker": null,
+  "moreTransitionsAvailable": false
+}
```
Comment (natke) [inline src/test/resources/data/ItemDoorTransitionResponse.json]: Is there a weird character here?

Comment (phil-impinj) [inline src/test/resources/data/ItemDoorTransitionResponse.json]: I don't know what you are referring to as a weird character.

**Comment by phil-impinj** [comment]
Version is bumped.

**Pull request merged**